docs/validate-package.md

```diff
@@ -0,0 +1,90 @@
+# How do I validate the package?
+
+Once you've [created your operator's package manifests](packaging-an-operator.md), you will want to ensure that your package is valid and in the correct format. To ensure this, you should take several steps to ensure that your package can be used to install your operator in OLM. We will publish the package into a catalog, install that catalog onto a Kube cluster, and then install the operator onto that cluster. If all of that succeeds and your operator is behaving as expected, your package is valid.
+
+## Linting
+
+As a quick check to statically verify that your package is valid, you can use the `operator-courier` project to check a few basic things:
+
+```
+$ pip3 install operator-courier
+$ operator-courier verify manifests/my-operator-package
+```
+
+This will check the packaging format for some basic checks to ensure that your package is set up correctly.
+
+You can also check for some additional requirements to ensure that your operator will display properly on [OperatorHub.io](https://operatorhub.io/)
+
+```
+$ operator-courier verify --ui_validate_io manifests/my-operator-package
+```
+
+## Add your package to a catalog
+
+The [operator-registry project](https://github.com/operator-framework/operator-registry) defines a format for storing sets of operators and exposing them to make them available on a cluster. As part of adding your package to an operator-registry catalog, the operator-registry tools will verify that your operator is packaged properly ("Does it have a valid CSV of the correct format?", "Does my CRD properly reference my CSVs?", etc.). The simplest way to test that your package can be added to a catalog is by actually attempting to create a catalog that includes your operator.
```

Comment:
nit: down the road we should replace the catalog/registry intro w/ a link to an independent doc.

Reply:
I think including a link here (rather than just to the operator-registry repo) would be valuable here, but I don't think I see a problem with having a quick intro into how the registry relates to package validation and why you should care about it. We should be careful about the ratio of leaving a trail of breadcrumbs vs overexplaining and repeating ourselves in these docs imo. But I don't think that's a trivial problem to solve.

Reply:
Yeah, it's definitely a balance. Just a passing thought.

> But I don't think that's a trivial problem to solve.

agreed